packages/repository/src/__tests__/unit/repositories/relations-helpers/resolve-has-many-through-metadata.unit.ts

```diff
@@ -28,6 +29,33 @@ describe('HasManyThroughHelpers', () => {
       expect(result).to.containEql({categoryId: 1});
     });
   });
+  context('createFkValues', () => {
+    it('can return the target fk value of a given through instance', () => {
+      const resolved = resolvedMetadata as HasManyThroughResolvedDefinition;
+      const through1 = createCategoryProductLink({
+        id: 1,
+        categoryId: 2,
+        productId: 9,
+      });
+      const result = createFkValues(resolved, through1);
```

Comment:
Nitpick: 

I find the name `resolved` as confusing because it does not say what is being resolved (and what kind of data are we passing in the first argument). Have you considered using `relationMeta` or something similar?